src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -588,3 +591,59 @@ components:
           type: string
           description: 'Self diagnosed health'
           example: 'ok'
+
+  examples:
+    PostDSRecord:
+      value:
+        id: 'ds.radio:oai:du:c124a602-7fe6-470e-a484-c771db44aee1'
+        origin: 'ds.radio'
+        recordType: 'DELIVERABLEUNIT'
+        data:
+          <xip:DeliverableUnit>
+          <Metadata>
+          <PBCoreDescriptionDocument:PBCoreDescriptionDocument>
+          <exampleTitle>Radioavisen</exampleTitle>
+          <exampleDescription>This is an example description of a news broadcast.</exampleDescription>
+          </PBCoreDescriptionDocument:PBCoreDescriptionDocument>
+          </Metadata>
+          </<xip:DeliverableUnit>>
+
+    GetDSRecordWithChildren:
+      value:
+        id: "ds.radio:oai:du:c124a602-7fe6-470e-a484-c771db44aee1"
+        orgid: "null"
+        id_error: false
+        origin: "ds.radio"
+        recordType: "DELIVERABLEUNIT"
+        deleted: false
+        data: |
+          <xip:DeliverableUnit>Metadata for DeliverableUnit</xip:DeliverableUnit>
+        cTime: 1701180992527000
+        mTime: 1701262548465000
+        cTimeHuman: "2023-11-28 15:16:32+0100"
+        mTimeHuman: "2023-11-29 13:55:48+0100"
+        parentId: "null"
+        childrenIds: [ "ds.radio:oai:man:b39d4884-66d2-4e49-b164-0d5c7e05ca49" ]
+        parent: "null"
+        children: "<xip:Manifestation>Metadata for Manifestation</xip:Manifestation"
```

Comment:
Doubt this is correct. The child(array) is a again list of full records and not just the metadata field.

Reply:
You are correct! I will fix this